src/tr/mangatilkisi/src/eu/kanade/tachiyomi/extension/tr/mangatilkisi/MangaTilkisi.kt

```diff
@@ -0,0 +1,12 @@
+package eu.kanade.tachiyomi.extension.tr.mangatilkisi
+
+import eu.kanade.tachiyomi.multisrc.madara.Madara
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class MangaTilkisi : Madara(
```

Comment:
Related to #4762?

Reply:
yes

Reply:
Then you should remove that extension in this PR. Since it's a different theme there's no point in keeping the id.